roles/patch/tasks/main.yml

```diff
@@ -39,6 +39,24 @@
   become: true
   become_user: root
 
+- name: Bug 31748584 resolution - MOS (Doc ID 2283767.1) Case3a
```

Comment:
Suggested wording:

Clean up old files in OPatch directory (MOS Doc ID 2283767.1, case 3a)

Reply:
Updated name of task as suggested.